winui/AvatarView/content-types.md

```diff
@@ -1,10 +1,267 @@
 ---
 layout: post 
 title: Content Types in WinUI AvatarView control | Syncfusion
-description: Learn what are the content types available and how to set value for the content types in WinUI AvatarView control.
+description: Learn about the available content types and how to set values for them in the WinUI AvatarView control.
 platform: WinUI
 control: AvatarView
 documentation: ug
 ---
 
-# Content Types in WinUI AvatarView
\ No newline at end of file
+# Content Types in WinUI AvatarView
+
+The AvatarView control supports five different types of content: `Default`, `Initials`, `CustomImage`, `AvatarCharacter`, and `GroupView`.
+
+## Default
+
+Default content type is used for displaying the `Avatar1` character image when initializing without the initials, custom image, or group view types.
+
+{% tabs %}
+{% highlight xaml %}
+
+<Grid>
+<syncfusion:SfAvatarView />
+</Grid>
+
+{% endhighlight %} 
+{% highlight C# %}
+
+// Creating an instance of the AvatarView control.
+SfAvatarView avatarView = new SfAvatarView();
+           
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control](avatarview_images/winui_avatarview.png)
+
+## Initials 
+
+Initials content type is used to show initials to the `AvatarView`. When using the `ContentType` as initials, you need to set the initial character using the following properties:
+
+* `InitialsType` - Defines the type of characters to be displayed.  
+* `AvatarName` - Defines the initials to be displayed.
+
+### InitialsType
+
+The `InitialsType` contains the following two types:
+
+* SingleCharacter
+* DoubleCharacter
+
+You must set the `AvatarName` string property for displaying the initials value in the AvatarView.
+
+**SingleCharacter**
+
+The `SingleCharacter` is used for displaying the first character in the string you have set in the `AvatarName` property.
+
+{% tabs %}
+{% highlight xaml %}
+
+<syncfusion:SfAvatarView ContentType="Initials"
+                         AvatarSize="ExtraLarge"
+                         AvatarName="Alex"
+                         Background="Bisque">
+</syncfusion:SfAvatarView>
+
+{% endhighlight %}
+{% highlight c# %}
+             
+SfAvatarView avatarView = new SfAvatarView();
+avatarView.AvatarName = "Alex";
+avatarView.ContentType = AvatarContentType.Initials;
+avatarView.AvatarSize = AvatarSize.ExtraLarge;
+avatarView.Background = new SolidColorBrush(Colors.Bisque);
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with SingleCharacter](avatarview_images/winui_initials_avatarview.png)
+
+**DoubleCharacter**
+
+The `DoubleCharacter` is used for displaying a two-characters text you have set in the `AvatarName` property. If the initials consist of a single word, it will show the first and last letters of that word. If the initials contain two or more words, it will display the first letter of the first word and the first letter of the second word.
+
```

Comment:
@MadhumithanJayapathi The double character initials type is used to display a two-character text as avatar that has been set in the AvatarName property. If the `AvatarName` consist of a single word, the first and last letters of that word will be displayed. However, if the `AvatarName` contains two or more words, the first letter of the first word and the first letter of the second word will be displayed.

Reply:
@Manivannan-E Changed the content of the double character as mwntioned.